config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -10,7 +10,7 @@ presubmits:
     - ^main$
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231206-f7b83ffbe6-1.28
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231206-f7b83ffbe6-1.29
```

Comment:
is this a valid tag?
normally sig testing runs a script that batch updates all files.

Reply:
same problem for https://github.com/kubernetes/test-infra/pull/31451

see config/jobs/kubernetes/sig-release/release-branch-jobs/1.29.yaml in
https://github.com/kubernetes/test-infra/pull/31405

```
gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231208-8b9fd88e88-1.29
```

Reply:
Yes, it is used a lot in test-infra already 🤔 :

https://cs.k8s.io/?q=gcr.io%2Fk8s-staging-test-infra%2Fkubekins-e2e%3Av20231206-f7b83ffbe6-1.29&i=nope&files=&excludeFiles=&repos=

I think the batch update always only updates the version used, so for all `*1.28`, it will bump. But to go from `*1.28` to `*1.29` we explicitly have to bump on our own.

Reply:
you could just bump to `gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231208-8b9fd88e88-1.29` manually here and other PRs, and it will be valid, IMO.

Reply:
Ah, now I got your point. Thanks for the hint!